--- config.h	2021-08-15 12:14:47.437118480 -0400
+++ config.h	2021-08-15 12:14:47.265107167 -0400
@@ -1,5 +1,5 @@
 /* See LICENSE file for copyright and license details. */
-#include <X11/XF86keysym.h> /* for volume keys */
+#include <X11/XF86keysym.h> /* for volume and brightness keys */
 
 /* appearance */
 static const unsigned int borderpx  = 1;        /* border pixel of windows */
@@ -22,6 +22,8 @@
 static const char *upvol[]   = { "/usr/bin/pulsemixer", "--change-volume", "+5", NULL };
 static const char *downvol[] = { "/usr/bin/pulsemixer", "--change-volume", "-5", NULL };
 static const char *mutevol[] = { "/usr/bin/pulsemixer", "--toggle-mute", NULL };
+static const char *upbrightness[]   = { "/usr/bin/light", "-A", "5", NULL };
+static const char *downbrightness[] = { "/usr/bin/light", "-U", "5", NULL };
 static const Bool viewontag = True; /* Switch view on tag switch */
 
 /* tagging */
@@ -113,6 +115,8 @@
 	{ 0,              XF86XK_AudioLowerVolume, spawn,          {.v = downvol } },
 	{ 0,              XF86XK_AudioMute,        spawn,          {.v = mutevol } },
 	{ 0,              XF86XK_AudioRaiseVolume, spawn,          {.v = upvol   } },
+	{ 0,              XF86XK_MonBrightnessDown, spawn,         {.v = downbrightness } },
+	{ 0,              XF86XK_MonBrightnessUp,   spawn,         {.v = upbrightness   } },
 };
 
 /* button definitions */
