--- config.h	2021-07-11 12:06:30.730186538 -0400
+++ config.h	2021-07-11 12:06:30.722186057 -0400
@@ -56,8 +56,8 @@
 
 /* commands */
 static char dmenumon[2] = "0"; /* component of dmenucmd, manipulated in spawn() */
-static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_cyan, "-sf", col_gray4, NULL };
-static const char *termcmd[]  = { "st", NULL };
+static const char *dmenucmd[] = { "/home/dsicilia/dev/suckless/dmenu/latest/dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_cyan, "-sf", col_gray4, NULL };
+static const char *termcmd[]  = { "/home/dsicilia/dev/suckless/st/latest/st", NULL };
 
 static Key keys[] = {
 	/* modifier                     key        function        argument */
