--- config.h	2021-07-13 23:16:06.007844489 -0400
+++ config.h	2021-07-13 23:16:05.943845621 -0400
@@ -1,4 +1,5 @@
 /* See LICENSE file for copyright and license details. */
+#include <X11/XF86keysym.h> /* for volume keys */
 
 /* appearance */
 static const unsigned int borderpx  = 1;        /* border pixel of windows */
@@ -17,6 +18,9 @@
 	[SchemeNorm] = { col_gray3, col_gray1, col_gray2 },
 	[SchemeSel]  = { col_gray4, col_cyan,  col_cyan  },
 };
+static const char *upvol[]   = { "/usr/bin/pulsemixer", "--change-volume", "+5", NULL };
+static const char *downvol[] = { "/usr/bin/pulsemixer", "--change-volume", "-5", NULL };
+static const char *mutevol[] = { "/usr/bin/pulsemixer", "--toggle-mute", NULL };
 
 /* tagging */
 static const char *tags[] = { "1", "2", "3", "4", "5", "6", "7", "8", "9" };
@@ -96,6 +100,10 @@
 	TAGKEYS(                        XK_9,                      8)
 	{ MODKEY|ShiftMask,             XK_q,      quit,           {1} }, /* restart only */
 	{ MODKEY|ControlMask|ShiftMask, XK_q,      quit,           {0} }, /* really quit dwm */
+  /* With 0 as modifier, you are able to use the keys directly. */
+	{ 0,              XF86XK_AudioLowerVolume, spawn,          {.v = downvol } },
+	{ 0,              XF86XK_AudioMute,        spawn,          {.v = mutevol } },
+	{ 0,              XF86XK_AudioRaiseVolume, spawn,          {.v = upvol   } },
 };
 
 /* button definitions */
