--- config.h	2021-10-08 09:49:45.389111571 -0400
+++ config.h	2021-10-08 09:49:45.333108089 -0400
@@ -24,6 +24,7 @@
 static const char *mutevol[] = { "/usr/bin/pulsemixer", "--toggle-mute", NULL };
 static const char *upbrightness[]   = { "/usr/bin/light", "-A", "5", NULL };
 static const char *downbrightness[] = { "/usr/bin/light", "-U", "5", NULL };
+static const char *touchpadtoggle[] = { "/home/dsicilia/dev/utilities/mouse/toggle-touchpad-tapping.sh", NULL };
 static const Bool viewontag = True; /* Switch view on tag switch */
 
 /* tagging */
@@ -117,6 +118,7 @@
 	{ 0,              XF86XK_AudioRaiseVolume, spawn,          {.v = upvol   } },
 	{ 0,              XF86XK_MonBrightnessDown, spawn,         {.v = downbrightness } },
 	{ 0,              XF86XK_MonBrightnessUp,   spawn,         {.v = upbrightness   } },
+	{ 0,              XF86XK_TouchpadToggle,    spawn,         {.v = touchpadtoggle } },
 };
 
 /* button definitions */
