--- config.h	2021-07-11 09:08:00.604970733 -0400
+++ config.h	2021-07-11 11:16:58.953149349 -0400
@@ -5,8 +5,8 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
-static int borderpx = 2;
+static char *font = "Literation Mono Powerline:spacing=100:size=9.5:antialias=true:autohint=true";
+static int borderpx = 0;
 
 /*
  * What program is execed by st depends of these precedence rules:
